synsets/03/00/86/ravesnik.xml

```diff
@@ -12,11 +12,12 @@
       steen:type="2"
       steen:same="v pl"
     >
-      råvěsnik
+      råvesnik
     </lemma>
   </synset>
   <synset lang="en">
     <lemma>coeval</lemma>
+    <lemma annotation="of same age">peer</lemma>
```

Comment:
Myslim, da primětka tut ne je potrěbna: to može značiti _peer_ i v obćem smyslu (kako i jest v dvoh izvedenyh iz njego slov).